src/opm/parser/eclipse/EclipseState/Schedule/VFPProdTable.cpp

```diff
@@ -525,6 +525,35 @@ void VFPProdTable::convertGFRToSI(const GFR_TYPE& type,
 }
 
 
+void VFPProdTable::convertAlqToSI(const ALQ_TYPE& type,
+                                  std::vector<double>& values,
+                                  const UnitSystem& unit_system) {
+    double scaling_factor = 1.0;
+    switch (type) {
+        case ALQ_IGLR:
+            scaling_factor = unit_system.parse("GasSurfaceVolume/LiquidSurfaceVolume/Time").getSIScaling();
```

Comment:
This does not work; the unit system parser is *really* quite primitive - at only allows one `/` literal, and that should be interpreted as one large divsion sign - i.e.:
```
                                                             GasSurfaceVolume
GasSurfaceVolume / LiquidSurfaceVolume * Time  ==>     ---------------------------
                                                        LiquidSurfaceVolume * Time   
```   
```